### ISSUE TITLE
Support aria-haspopup attribute on Element

### ISSUE THREAD

**Issue by alansemenov** [enonic/lib-admin-ui#3650]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-haspopup

Should be enum of supported values
